### PR TITLE
LRQA-69807 Forms#AddFormToUserSiteAndViewNoDuplicates Test fix

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-69807

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase [hunk 1]
```diff
@@ -213,7 +213,11 @@ definition {
 
 		Navigator.openSiteURL(siteName = "Form Site");
 
-		FormsAdmin.addForm();
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase:216]: @leticiachiu, please see this [comment](https://github.com/liferay-forms/liferay-portal/pull/894#discussion_r703070100) and think if we can apply the same suggestion here. :wink:

**Comment by leticiachiu** [comment]
@natocesarrego I was investigating this test fix and I don't think that keeping this macro is worthy on this specific case. To adapt, I had to change a lot the code and just for one test (because I can't use the parameter "Guest" or "Global" so is harder to get the groupId and groupName) 😅😅

**Comment by natocesarrego** [comment]
No problem, @leticiachiu! :smile: Thanks for your investigation. :wink:

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**